README.markdown

```diff
@@ -30,7 +30,7 @@ Note: On OS X, if you have/had a "system" node installed and want to install mod
 
 Homebrew installation is not supported. If you have issues with homebrew-installed `nvm`, please `brew uninstall` it, and install it using the instructions below, before filing an issue.
 
-Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm_update` to update.
+Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm upgrade` to upgrade.
```

Comment:
Wouldn't them both using the same command be a benefit? If `nvm upgrade` gets added to `nvm` in the future I can hand upgrade functionality over to `nvm` and it'll be seamless for current users of `zsh-nvm`. What do you think?